### ISSUE TITLE
vcard_attachments : display all the content of the card

### ISSUE THREAD

**Issue by dominiquefournier** [roundcube/roundcubemail#6561]
Hi
It should be super if all the content of the vcard can be displayed. I suggests this code :

**Comment by alecpl** [comment]
I don't think we should display it "inline", but maybe we could add "Preview" button.

**Comment by dominiquefournier** [comment]
It would be really cool ! Maybe just add a javascript to hide the block and display it when the user click on it
